### PR TITLE
docs(zh-CN): add zh-CN resource reference

### PR DESCRIPTION
**Pull request by simeng-li**
add zh-CN resource reference doc.

image

@logto-io/eng

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+    redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+    code=10esc29BWC2qZB0acc9v8zAv9ltc2pko105tQauZ
+    resource=https%3A%2F%2Flogto.test.dev%2Fusers
+```
+
+请求成功后，Logto 将颁发一个 `access_token` 访问令牌，其受众目标仅限于该被请求的接口或资源。
+
+### 3. API Resource 请求
+
+每一个由客户端发起面向一个 API resource 访问的请求头中将携带一个由 Logto 颁发，带有受众，授权，时效等信息的加密 `access_token` 令牌。
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md:75]: maybe better?
```suggestion
每一个由客户端发起面向一个 API resource 访问的请求头中将携带一个由 Logto 颁发，带有受众，授权，时效等信息的 `access_token` 加密令牌。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/fragments/_api_resource_schema.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      </td>
+      <td>否</td>
+    </tr>
+  </tbody>
+</table>
+
+:::caution
+一个带有过期令牌的请求将被视为无效请求。
+这个设置是为了防止你的接口长期暴露在一个可能被劫持的令牌下。
+该数值越高，一个令牌的有效生命周期就越长，你的接口就将面临在更大的风险
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/fragments/_api_resource_schema.mdx:47]:
```suggestion
该数值越高，一个令牌的有效生命周期就越长，你的接口就将面临在更大的风险。
```

File: docs/docs/recipes/protect-your-api/README.md [hunk 3]
```diff
@@ -7,26 +7,26 @@ import TabItem from '@theme/TabItem';
 
 # ⚔️ Protect your API
 
-Along with a fluent user sign-in experience, by default, Logto comes with the [API resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API resource authorization guard of your own using Logto.
+Along with a fluent user sign-in experience, by default, Logto comes with the [API Resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API Resource authorization guard of your own using Logto.
```
Comment (gao-sun) [inline docs/docs/recipes/protect-your-api/README.md:10]: the commit message and pr title looks verbose. in docs we just have two scopes now: `(en-US)` and `(zh-CN)`

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 4]
```diff
@@ -1 +1,88 @@
...
+
+- 一个有效的资源标识，用来表示被请求方的目标接口或者资源。
+- 它的值**必须**是一个完整的 URI 格式。
+- 该 URI **不能**包含任何 fragment 片段。
+- 该 URI **不应**包含任何 query 片段。
+- 该 URI **应**尽可能的具体。用于表示一个或一组完整的接口或资源。
+
+在实际应用中，客户端可能仅知道与其交互的应用程序或者被请求资源的根地址。这时就可以用它来作为 Resource Indicator， 用来代表所有该地址下的接口和资源
+
+例如： Logto 提供的管理接口 URI 地址
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md (hunk 1 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 5]
```diff
@@ -1 +1,88 @@
...
+
+默认初始状态下，该 URI 就会被作为一个 API Resource 注册在 Logto 侧。 所有该地址下的管理接口都会受到 Logto 服务器的保护。
+
+### Logto API Resource 的数据格式
+
+<APIResourceSchema />
+
+## API Resource 是如何运作的
+
+### 1.授权请求
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 6]
```diff
@@ -1 +1,88 @@
...
+
+Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiYXVkIjoiaHR0cHM6Ly9sb2d0by5kZXYvYXBpL3VzZXJzIiwiaXNzIjoiaHR0cHM6Ly9sb2d0by5kZXYvb2lkYyIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMiwiZXhwIjoxNTE2MzI1NDIyfQ.PjIJl00YNC84EPNYLEdpEEAdAxqhekCYhFEckvRokek
+
+```
+
+:::tip
+请参照我们的 [**API 资源保护**](../../recipes/protect-your-api/README.md) 指南，添加令牌验证到你的服务器接口端，是你的私有资源得到更好的保护。
+:::
+
+Logto 遵循标准的 OAuth 2.0 令牌授权协议来保护你的数据资源. 你可以访问 OAuth 2.0 的[官方文档(https://datatracker.ietf.org/doc/html/rfc6749#section-1.3.1)，以获取跟多细节.
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: docs/docs/recipes/protect-your-api/README.md (hunk 3 above)
Comment (simeng-li) [reply docs/docs/recipes/protect-your-api/README.md]: updated and rebase the commit message

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 7]
```diff
@@ -1 +1,88 @@
...
+Logto, 作为一个授权服务器，旨在为大量不同的资源和接口提供权限服务。我们将会根据用户所声明的 API 资源标识来生成和颁发带有特定内容、类型，和受众目标的加密令牌。
+
+将授权令牌及对应的权限校验应用到你的请求当中，可以更好的保护你的私有资源不被匿名身份者访问或者攻击。
+
+## 定义
+
+### Resource Indicator
+
+- 一个有效的资源标识，用来表示被请求方的目标接口或者资源。
+- 它的值**必须**是一个完整的 URI 格式。
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md:22]:
```suggestion
- 它的值 **必须** 是一个完整的 URI 格式。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 8]
```diff
@@ -1 +1,88 @@
...
+- 该 URI **不能**包含任何 fragment 片段。
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md:23]:
```suggestion
- 该 URI **不能** 包含任何 fragment 片段。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 9]
```diff
@@ -1 +1,88 @@
...
+- 该 URI **不应**包含任何 query 片段。
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md:24]:
```suggestion
- 该 URI **不应** 包含任何 query 片段。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md [hunk 10]
```diff
@@ -1 +1,88 @@
...
+- 该 URI **应**尽可能的具体。用于表示一个或一组完整的接口或资源。
```
Comment (darcyYe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md:25]:
```suggestion
- 该 URI **应** 尽可能的具体。用于表示一个或一组完整的接口或资源。
```

**Review by darcyYe — APPROVED** [review]

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md (hunk 7 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md (hunk 8 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md (hunk 9 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md (hunk 10 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/resources/README.md]: updated

**Pull request merged**